app/src/main/java/org/jellyfin/mobile/utils/QualityOptions.kt

```diff
@@ -0,0 +1,180 @@
+// Taken and adapted from https://github.com/jellyfin/jellyfin-web/blob/ebb4b050817d4d1920ec28209dee0539410c06ef/src/components/qualityOptions.js
+
+package org.jellyfin.mobile.utils
+
+import org.json.JSONArray
+import org.json.JSONException
+import org.json.JSONObject
+import java.text.DecimalFormat
+import java.text.StringCharacterIterator
+
+object QualityOptions {
+    fun getVideoQualityOptions(options: String): String {
+        with(JSONObject(options)) {
+            return getVideoQualityOptions(
+                currentMaxBitrate = optInt("currentMaxBitrate"),
+                isAutomaticBitrateEnabled = optBoolean("isAutomaticBitrateEnabled"),
+                videoWidth = optInt("videoWidth", 4096),
+                videoHeight = optInt("videoHeight"),
```

Comment:
Why 4096? Why no default value for the height? This should be documented somewhere.

Reply:
No idea, ask jf-web why

https://github.com/jellyfin/jellyfin-web/blob/e7521162097fdb5b2731510a634339be9a690add/src/components/qualityOptions.js#L14

Reply:
Oh.. :(
I'll bring this up with the web team then.